examples/python/pincell_depletion/run_depletion.py

```diff
@@ -16,7 +16,7 @@
 time_step = 1*24*60*60 # s
 final_time = 5*24*60*60 # s
 time_steps = np.full(final_time // time_step, time_step)
-
+print(time_steps)
```

Comment:
Remove

Reply:
Left a few hasty debug statements in here... sorry!